.github/workflows/build.yaml

```diff
@@ -279,6 +279,43 @@ jobs:
           name: macOS-dmg
           path: Freeciv21-*.dmg
 
+  fedora:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule'
+    container: fedora:latest
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo dnf install -y \
+            git \
+            cmake \
+            ninja-build \
+            python \
+            gettext \
+            qt5-qtbase-devel \
+            qt5-qtsvg-devel \
+            kf5-karchive-devel \
+            lua-devel \
+            sqlite-devel \
+            SDL2_mixer-devel \
+            readline-devel \
+            zlib-devel \
+            libunwind-devel \
+            elfutils-libs \
+            python3-sphinx
```

Comment:
If not needed in instructions, prolly not here either.

Reply:
Good catch :+1: